ci/scripts/run-benchmarks.sh

```diff
@@ -20,7 +20,7 @@ export PG_VERSION=15
 export GITHUB_OUTPUT=/dev/null
 export PGDATA=/etc/postgresql/$PG_VERSION/main/
 
-echo "port = 5432" >> $PGDATA/postgresql.conf
+echo "port = 5432" >> ${PGDATA}postgresql.conf
 # Enable auth without password
 echo "local   all             all                                     trust" >  $PGDATA/pg_hba.conf
 echo "host    all             all             127.0.0.1/32            trust" >>  $PGDATA/pg_hba.conf
```

Comment:
note that if we keep the current value of `PGDATA` , these two lines need to be updated to remove the slash after  `$PGDATA`